Source/Response.swift

```diff
@@ -77,6 +88,7 @@ extension Response: CustomDebugStringConvertible {
         output.append(response != nil ? "[Response]: \(response!)" : "[Response]: nil")
         output.append("[Data]: \(data?.length ?? 0) bytes")
         output.append("[Result]: \(result.debugDescription)")
+        output.append("[Timeline]: \(timeline.debugDescription)")
```

Comment:
There's a new line between this and the `[Result]` output. Perhaps there's an extra newline in `result.debugDescription`? Trivial, but didn't quite match the rest of the output.